src/styles/components/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  width: 100px;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-around;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), using `space-around` is not recommended due to inconsistent spacing behavior. Consider using `justify-content: space-between` for consistent spacing.